README.md

```diff
@@ -10,35 +10,85 @@ The intent of the project is to leverage the collective knowledge of the dispara
 
 ### Prerequisites
 
-* A newer version of Python 3, preferably 3.10 or greater. To confirm on your system, run:
+- A newer version of Python 3 (preferably 3.10+)
+- [Node.js](https://nodejs.org/en) (v18 or later)
```

Comment:
You have instructions on how to install pnpm, would be nice to also have something on how to install Node.js in here too for consistency